services/heartrate.md

```diff
@@ -0,0 +1,18 @@
+#  Heart Rate
+
+A sensor approximating the heart rate. 
+
+**JACDAC is not suitable for medical devices and should NOT be used in any kind of device to diagnose or treat any medical conditions.**
+
+    identifer: 0x166c6dc4
+    extends: _sensor
+
+## Registers
+
+    ro heart_rate: i8.8 bpm @ reading
+
+The estimated heart rate. Negative value when the sensor cannot sense the rate.
```

Comment:
There is more information because the sensor knows if a finger is present for example. But it's not really like a intensity register as it is readonly.

Reply:
maybe we should define status code values? I don't like negative for error - you have then deal with that in your app